charts/service-binding-operator/templates/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: service-binding-controller-manager
+  name: service-binding-operator
```

Comment:
hum... I do think it is strange that this works right now, the namespace resource happens to come before all others. I would prefer we could hardcore the namespace, but I agree with @pedjak lets do it the helm way. User should have the option to install in current namespace or create one at install time. No need to put in values file though.

Reply:
It is working as of now because we are creating that namespace and have all resources in that namespace. Actually the helm release is happening in a different namespace (where ever the user has context set) ,but the chart workloads happen to be in service-binding-operator ns .
As a workaround i will be installing the chart as :
helm install sbo ./Projects/service-binding-operator/charts/service-binding-operator  --namespace service-binding-operator --create-namespace.
The --create-namespace param seems to be a bool flag instead of string and i believe then i can refernce the namespace field in the template with {{.Release.Nmaespace}}

Reply:
> It is working as of now because we are creating that namespace and have all resources in that namespace. Actually the helm release is happening in a different namespace (where ever the user has context set) ,but the chart workloads happen to be in service-binding-operator ns .

The helm release (actually the secret keeping all information about the installed chart) should be placed in the same namespace as the operator - otherwise, the user experience will become very strange - how you would ever find where your Helm release is - if it is just based on the fact what was your context at the time when the chart was installed?